utils/gridadmin.py

```diff
@@ -458,21 +460,15 @@ def save(
 
             for node_id in [node1_id, node2_id]:
                 if node_id == -9999:
-                    try:
-                        line.AddPoint_2D(
-                            self.node_data["coordinates"][0][node1_id] + 5,
-                            self.node_data["coordinates"][1][node1_id] + 5,
-                        )
```

Comment:
Ik geloof dat de `+5` aan de coördinaten wel bewust is gedaan, om te zorgen dat twee punten niet overlappen.

Reply:
Goeie! Compleet over het hoofd gezien.
Ik zet de oude code terug (wel met de logger.debug/logger.exception wijziging).